include/attribute-based/bsw/ciphertext-policy/BSWCiphertextPolicyAttributeBasedEncryptionPolynom.h

```diff
@@ -0,0 +1,33 @@
+#ifndef __CRYPTID_CRYPTID_POLYNOM_H
+#define __CRYPTID_CRYPTID_POLYNOM_H
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "gmp.h"
+
+#include "attribute-based/bsw/ciphertext-policy/BSWCiphertextPolicyAttributeBasedEncryptionUtils.h"
+#include "attribute-based/bsw/ciphertext-policy/BSWCiphertextPolicyAttributeBasedEncryptionDefines.h"
+#include "util/Random.h"
+#include "util/Status.h"
+#include "attribute-based/bsw/ciphertext-policy/BSWCiphertextPolicyAttributeBasedEncryptionMasterKey.h"
+
+typedef struct BSWCiphertextPolicyAttributeBasedEncryptionPolynom
+{
+	int degree;
+	struct BSWCiphertextPolicyAttributeBasedEncryptionPolynomExpression** children;
```

Comment:
You should switch up the order of the two structs here and delete the `struct` keyword from this line